dll/cpl/desk/lang/en-GB.rc

```diff
@@ -184,9 +184,9 @@ BEGIN
     LTEXT "", IDC_FONTSIZE_CUSTOM, 14, 46, 223, 12
     GROUPBOX "Compatibility", -1, 7, 67, 237, 97
     LTEXT "Some programs operate improperly if you do not restart your computer after you change colour settings.\n\nAfter I change colour settings", -1, 12, 78, 228, 34
-    AUTORADIOBUTTON "&Restart the computer before applying the new colour settings.", IDC_RESTART_RB, 14, 116, 228, 10
-    AUTORADIOBUTTON "Apply the new colour settings without restarting.", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10
-    AUTORADIOBUTTON "Ask me before applying the new colour settings.", IDC_ASKME_RB, 14, 146, 228, 10
+    AUTORADIOBUTTON "&Restart the computer before applying the new colour settings", IDC_RESTART_RB, 14, 116, 228, 10
+    AUTORADIOBUTTON "Apply the new colour settings without restarting", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10//FIXME: add accel
+    AUTORADIOBUTTON "Ask me before applying the new colour settings", IDC_ASKME_RB, 14, 146, 228, 10//FIXME: add accel
```

Comment:
As in en-US of course:
```suggestion
    AUTORADIOBUTTON "Apply the &new colour settings without restarting", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10
    AUTORADIOBUTTON "As&k me before applying the new colour settings", IDC_ASKME_RB, 14, 146, 228, 10
```
(and in any other translation file here where these sentences stayed in english, for example in the el-GR.rc above)

Reply:
I am aware of how to proceed when the english one is found, I just lacked the en-US pic until Doug posted it.

Reply:
Resolved.